db/seeds.d/07-provisioning_templates.rb

```diff
@@ -17,13 +19,14 @@
       end
     else
       next if SeedHelper.audit_modified? ProvisioningTemplate, input[:name]
-      input[:default] = true
-
       t = ProvisioningTemplate.create({
-        :snippet  => false,
-        :template => contents,
-        :locked => true
-      }.merge(input))
+                                        :template => contents,
+                                        :locked => true,
+                                        :default => true,
+                                        :name => input[:name],
+                                        :template_kind => input[:template_kind],
```

Comment:
Make sure the `template_kind` passed to in tests exists, either via FactoryGirl or fixtures. Tests failing mention `ActiveRecord::InvalidForeignKey: PG::ForeignKeyViolation: ERROR:  insert or update on table "templates" violates foreign key constraint "config_templates_template_kind_id_fk"... Key (template_kind_id)=(983253651) is not present in table "template_kinds".` which makes me think here's the issue